src/test/java/net/minecraftforge/debug/chat/ClientCommandTest.java

```diff
@@ -104,6 +106,11 @@ private void init(RegisterClientCommandsEvent event)
                                 .executes((context) -> {
                                     context.getSource().sendSuccess(Component.literal("Executed command"), false);
                                     return 1;
+                                }))
+                        .then(Commands.literal("screenpatch")
+                                .executes((stack) -> {
+                                    Minecraft.getInstance().setScreen(new PatchScreen(Component.literal("patch")));
```

Comment:
To avoid classloading problems with regards to the (implicit) cast of `PatchScreen` to `Screen`, please move this call to `Minecraft#setScreen` to a static method within `PatchScreen`, which this lambda calls.